app/components/candidate_interface/continuous_applications/application_summary_component.html.erb

```diff
@@ -16,7 +16,7 @@
         <%= title %>
       </h2>
       <ul class="govuk-summary-card__actions">
-        <% if application_choice.course_available? && !application_choice.unsubmitted? %>
+        <% if !application_choice.unsubmitted? %>
```

Comment:
Are we showing this when there are no more courses available? This was a requirement for a task I worked on for draft applications when the course becomes full.

Reply:
The candidate can't access the review page for unsubmitted applications anyway, even with this condition included:
<img width="745" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/47917431/8f518c6f-0fc5-400a-8a67-00fb62f16fef">

But if we leave it in it means a candidate wouldn't be able to view an already submitted application, if the course has become full at a later date. Since it's already submitted my understanding is that we'd be happy for them to view the application, as it would include the personal statement submitted for that choice cc @MaeveRoseveare

Reply:
Yeah agree, if the application is already submitted. We should show the 'View application' link. We will be doing this in the new designs we showed this morning, so makes sense to add it.

Reply:
Makes sense. We should tweak the double negation `if application_choice.submitted?`